src/components/DateTimeField/index.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import { InputText } from '@edx/paragon';
+import { getDateString, getTimeString } from '../../utils/index';
+import FieldLabel from '../FieldLabel';
+import { DATE_FORMAT } from '../../data/constants';
+
+class DateTimeField extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      date: getDateString(this.props.input.value) || '',
+      time: getTimeString(this.props.input.value) || '',
+    };
+    this.concatDateTime = this.concatDateTime.bind(this);
+    this.updateDate = this.updateDate.bind(this);
+    this.updateTime = this.updateTime.bind(this);
+  }
+
+  concatDateTime(date, time) {
+    const datetime = moment.utc(date + time, 'YYYY/MM/DD HH:mm').format(DATE_FORMAT);
+    this.props.input.onChange(datetime);
+  }
+
+  updateDate(value) {
+    const {
+      time,
+    } = this.state;
+    this.setState({ date: value });
+    this.concatDateTime(value, time);
+  }
+
+  updateTime(value) {
+    const {
+      date,
+    } = this.state;
+    this.setState({ time: value });
+    this.concatDateTime(date, value);
+  }
+
+  render() {
+    const {
+      dateName,
+      dateLabel,
+      timeName,
+      timeLabel,
+      helpText,
+      courseInReview,
+    } = this.props;
+
+    const {
+      date,
+      time,
+    } = this.state;
+
+    return (
+      <div className="row">
+        <div className="col-6">
+          <InputText
```

Comment:
It looks like we stripped out the 
```
          format={value => getDateString(value)}
          normalize={value => moment.utc(value).format(DATE_FORMAT)}
```

just making sure this is intentional

Reply:
Yeah, this is intentional! The date only needs to get normalized after the date & time strings are concatenated, and because we're using an InputText component rather than a Field, the "value" variable is the equivalent of "format"